WordPress/Classes/ViewRelated/Gutenberg/GutenbergImageLoader.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Gutenberg
+
+class GutenbergImageLoader: NSObject, RCTImageURLLoader {
+
+    public var post: AbstractPost
+
+    private lazy var mediaUtility: EditorMediaUtility = {
+        return EditorMediaUtility()
+    }()
+
+    public init(post: AbstractPost) {
+        self.post = post
+    }
+
+    func canLoadImageURL(_ requestURL: URL!) -> Bool {
+        return !requestURL.isFileURL
```

Comment:
I'm assuming `EditorMediaUtility` will only authenticate what's needed, but do we want to route every image loaded through the custom code? I'm thinking there might be a shared cache that makes this a good idea, but not sure what's involved in the decision

Reply:
So we have a local cache on MediaUtility but RN also has one. Do you think it will be better to only use it for private sites? or only for WP.com assets?

One of the advantages of using the MediaUtility is the ability to use Photon and WP image server resize.

Reply:
Photon seems like a good reason to use it for all images :shipit: